x-pack/legacy/plugins/integrations_manager/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "author": "Elastic",
+    "name": "epm",
+    "version": "8.0.0",
+    "private": true,
+    "license": "Elastic-License",
+    "dependencies": {
+      "react-markdown": "^4.2.2"
```

Comment:
Just to confirm, this is to get v4?

I see we have v3 in the main project & siem does something similar to get v4.

```
├─ react-markdown@3.6.0
└─ siem@8.0.0
   └─ react-markdown@4.0.8
```